sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -616,23 +616,76 @@ function withUserManagedIdentityCredential() {
 
 #### Authenticating in Azure Pipelines with service-connections
 
-This example demonstrates authenticating the `SecretClient` from the [@azure/keyvault-secrets][secrets_client_library] using the `AzurePipelinesServiceConnectionCredential` in an Azure Pipelines environment with service connections.
+##### Set up Federated Identity Credential in Azure Pipelines
+
+In order to set up FIC in Azure Pipelines, you can set up the Azure Resource Manager service connection as an [automatic recommended approach](https://learn.microsoft.com/azure/devops/pipelines/library/connect-to-azure?view=azure-devops#create-an-azure-resource-manager-service-connection-that-uses-workload-identity-federation). You can also create it manually either using a [user-assigned Managed Identity as FIC](https://learn.microsoft.com/azure/devops/pipelines/release/configure-workload-identity?view=azure-devops#set-a-workload-identity-service-connection-to-use-managed-identity-authentication) or using an [App Registration as FIC](https://learn.microsoft.com/azure/devops/pipelines/release/configure-workload-identity?view=azure-devops#set-a-workload-identity-service-connection-to-use-service-principal-authentication).
```

Comment:
To be a bit pedantic, we're not setting up the UAMI _as_ a FIC, we're setting up the UAMI to accept the ADO-issued token as a FIC. Federated Identity Credentials are just another credential type on an App Registration, along with Client Secret Credentials or Client Certificate Credentials.

Reply:
done